cpp/tests/utilities/column_utilities.cu

```diff
@@ -589,6 +594,7 @@ struct column_view_printer {
     //
     //  Implementation for strings, call special to_host variant
     //
+    if (col.size() == 0) { return; }
```

Comment:
```suggestion
    if (col.is_empty()) return;
```

Reply:
Ok, but it is absolutely not a safe practice to not use brackets on single-statement ifs.

Reply:
Yea, it was mostly about the `is_empty()`. The `{}` was less important, it is just my preference.